pages/test_analytics/test_suites.md

```diff
@@ -35,6 +35,25 @@ Alternatively, you can create [scheduled builds](/docs/pipelines/scheduled-build
 
 Test Analytics reviews the test results to detect flaky tests after every test run.
 
+## Run issues
+
+<%= image "run-issues.png", alt: "Screenshot of a run with issues displaying in a list, including flaky, slow and failures." %>
+
+Test Analytics will automatically detect issues per run. For each test, we currently detect up to three issues: flaky, slow and failure, listed in the following order of urgency:
```

Comment:
Technically its "number of issues, flaky, slow, failed", I wonder if we should clarify that?

Reply:
@KatieWright26 do you mean that it should be?

>  For each test, we detect flaky, slow and failed issues, along with the total number of issues found. These issues are listed in the following order of urgency:

Reply:
The confusion here is around the ordering, we say it is ordered 'flaky, slow and failure' where it is actually ordered 'total number of issues, flaky, slow and failure`

Something like:
For each test, we currently detect up to three issues: flaky, slow and failure. These issues are organised and presented in the following order based on their level of urgency, tests with the most amount of issues will be positioned at the top of the list.

Reply:
Still confused, almost there!

The system identifies these 3 issues for tests (the entire test file), and orders the list of test files with the one that has the most issues at the top, decending? eg.
**Worst spec**
`user_authentication_spec.rb`
* Total issues: 12
   * Slow: 4
   * Flaky: 4
   * Failed: 4

**Second worst spec**
`hello_user_spec.rb`
 * Total issues: 6
   * Slow: 2
   * Flaky: 2
   * Failed: 2

Reply:
It is per execution within test file, so not the whole file. The issues with either have a count of `1 | 0` so like your example:

**Worst spec**
`user_authentication_spec.rb:21`
Total issues: 3
Slow: 1
Flaky: 1
Failed: 1

**Second worst spec**
`user_authentication_spec.rb:2`
Total issues: 2
Slow: 1
Flaky: 0
Failed: 1

Do you think we should include that example in the doc?